socorro/signature/siglists_utils.py

```diff
@@ -43,6 +44,8 @@ def _get_file_content(source):
     with resource_stream(__name__, filepath) as f:
         for i, line in enumerate(f):
             line = line.strip()
+            if six.PY3:
+                line = line.decode('utf-8', 'strict')
```

Comment:
We can use `strict` here because if the signature lists are busted, then that's a bug and we should fix them. If there are problems, they'll show up in the tests in CI.